ap_ruby_sdk/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,13 @@
+# Contributor Code of Conduct
```

Comment:
I am not sure we should have generated readme files from idea. Lets remove and add when needed,